website/docs/r/elastic_beanstalk_application.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
   Provides an Elastic Beanstalk Application Resource
 ---
 
-# aws\_elastic\_beanstalk\_<wbr>application
+# aws_elastic_beanstalk_<wbr>application
```

Comment:
Nitpick: While you're touching these lines, could you remove these `<wbr>` things too?

Reply:
Sure thing! didn't catch my 👀